tests/bluetooth/qualification/ICS_Zephyr_Bluetooth_Host.pts

```diff
@@ -11717,12 +12000,16 @@
       </item>
       <item>
         <table>2</table>
-        <row>54</row>
+        <row>60</row>

```

Comment:
For some reason this wasn't enabled when I imported this file :s

Reply:
afair ICS are imported only on 'new' profile,  so you must first delete it from your workspace...